common/utils/EncryptionManager.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const crypto = require('crypto');
+const config = require('../config');
+const CONST = require('../constants');
+
+class EncryptionManager {
+
+  /**
```

Comment:
Are these JS docs helpful? We started adding these in the last couple of sprints, but most of the time they seem to self explanatory from function signatures. The reason am questioning this is, this makes our code have JSDoc on some files and not have in others. So i would prefer to have consistency.
Also, if we are not auto generating api documentation from JSDOC, then probably i would omit it.

Reply:
will remove it for this. ;)